WS gram pilot 2 ready

gkrajewski opened the pull request:
Według mnie działa i jest ok, ale sprawdźcie, proszę. I koniecznie przejrzyjcie historię commitów i krzyczcie, jeśli nie zgadzacie się z jakimiś moimi pomysłami.

@karolina-m Spójrz na przedostatniego commita, czy sensownie przeformułowałem info o e-kodach do Biedry :)

gkrajewski committed:
PL-WS grammar pilot 1 files duplicated

m-krysztofiak committed:
input files for gram-pilot 2.0

m-krysztofiak committed:
deixis correction
there are now two pages for deixis, so that an intermediate instruction can be entered

gkrajewski committed:
Grammar columns added & item_id sortable

gkrajewski committed:
Changed colname from "form" to "origin"
"form" is used in other parts of the app.

m-krysztofiak committed:
Merge pull request #301 from gkrajewski/pl-ws-gram-p2-items

Grammar columns added & item_id sortable

m-krysztofiak committed:
ws instruction made prettier

m-krysztofiak committed:
corrections translations

m-krysztofiak committed:
new gram items

m-krysztofiak committed:
new item type and category for inflection

m-krysztofiak committed:
new order and combined inflection

m-krysztofiak committed:
inflection type in translations

m-krysztofiak committed:
translations revert changes

m-krysztofiak committed:
remove combine-sentence condition
just for testing

m-krysztofiak committed:
inflection category

m-krysztofiak committed:
settings correct response type

m-krysztofiak committed:
minor corrections

m-krysztofiak committed:
long instruction for CDI3 part
this needs to be adjutsed, ideally we would have an interim instruction but I have to figure how to do it

gkrajewski committed:
Merge pull request #302 from gkrajewski/pl-ws-gram-p2

Up to date with pl-ws-gram-p2

m-krysztofiak committed:
new item order
includes additional item for nouns context

m-krysztofiak committed:
minor corrections
typos

gkrajewski committed:
Merge pull request #303 from gkrajewski/pl-ws-gram-p2

Up to date with pl-ws-gram-p2

gkrajewski committed:
Syntax items from CDI-III filtered
Only items with predicted difficulty for youngest CDI-III group of 24mo above .6 kept.

gkrajewski committed:
Inflection item removed from syntax items
A CDI-III question about locative plural removed, as we have a separate inflection section in WS.

gkrajewski committed:
Noun forms filtered
Details in a comment to issue #297

m-krysztofiak committed:
remove unnecessary header
przykłady wypowiedzi - we only have three sentences so it doesn't make much sense to keep this header

m-krysztofiak committed:
add catHeader alternatives2

m-krysztofiak committed:
demographic questions

m-krysztofiak committed:
instr to longText

m-krysztofiak committed:
demographic questions correction

m-krysztofiak committed:
minor correction issue #14

m-krysztofiak committed:
starting to talk added

m-krysztofiak committed:
new smocz. items

m-krysztofiak committed:
updated instructions

m-krysztofiak committed:
lexicon instruction

m-krysztofiak committed:
MS items first in "wypowiedzi"

m-krysztofiak committed:
wypowiedzi to zadania

gkrajewski committed:
Merge branch 'pl-ws-gram-p2' into pl-ws-gram-p2-items

gkrajewski committed:
Merge pull request #305 from gkrajewski/pl-ws-gram-p2-items

WS gram pilot 2 item selection

m-krysztofiak committed:
minor corrections
merged alternatives 1 and 2; changed html in adjectives_form; ms_items

gkrajewski committed:
Plural item improved
"kilka" > "więcej niż jedna"
Clearer examples
"już" deleted: consistent with other items and options to choose

gkrajewski committed:
Acc contexts removed
Acc use too broad to ask with specific verbs

gkrajewski committed:
Context items more generalised

gkrajewski committed:
Category adjective_form > adjective_adverb

gkrajewski committed:
Better adj/adv examples

gkrajewski committed:
Gender variation in adj examples

gkrajewski committed:
Gram labels & categories for adj/adv

m-krysztofiak committed:
help message correction
removed redundant help message for inflection
added help message for yesNowprost

m-krysztofiak committed:
demographic questions correction
na "Ty"

m-krysztofiak committed:
instructions sentences
per Ty

m-krysztofiak committed:
sentenes intruction correction
per Ty

m-krysztofiak committed:
sentences instruction correction

gkrajewski committed:
Merge pull request #310 from gkrajewski/pl-ws-gram-p2

GK branch up to date with WS gram pilot 2

gkrajewski committed:
Typo "zaznaczaczaj"

gkrajewski committed:
Demographic items adjustments
Removed PhD and some editing

gkrajewski committed:
"wyrazy" -> "słowa"

gkrajewski committed:
Minor edits

gkrajewski committed:
Overseas excluded
Upfront message & voivodeship Q moved first

gkrajewski committed:
No Część 1/2 in text and no first page in deixis

gkrajewski committed:
Header in deixis & help message in starting to talk

gkrajewski committed:
Formy & Konteksty as headers

gkrajewski committed:
Combine help corrected & combine item typo corrected in all Pl forms

gkrajewski committed:
Added norms for age restrictions

gkrajewski committed:
Plus or minus month to age range

m-krysztofiak committed:
setting with conditions

gkrajewski committed:
Merge pull request #311 from gkrajewski/pl-ws-gram-p2-gk

Various changes, modifications, corrections, improvements

gkrajewski committed:
Demographic additional help messages & tooltips

gkrajewski committed:
Translations properly sorted!

gkrajewski committed:
Adj/adv instruction more precise

gkrajewski committed:
Adj/adv items moved in the items file

gkrajewski committed:
Starting to talk items moved in the items file

gkrajewski committed:
Starting to talk items coherent in addressing parent

gkrajewski committed:
Settings with no conditions temporarily

gkrajewski committed:
Complexity alternatives categories renamed
And syntax items properly sorted in the items file

gkrajewski committed:
Direct questions category renamed

gkrajewski committed:
Complexity items properly sorted

gkrajewski committed:
Tentative gram labels for alternatives_simple

gkrajewski committed:
More simple alternatives from Smocz
Added all but four that either made no sense to me or redundant with inflection items

gkrajewski committed:
Complexity alternatives regrouped
Was 15+10, is 7+10+10+10: more items, more pages & first page shorter because of longText

gkrajewski committed:
First page alternatives longText reformatted
Paragraph that repeats on next pages put last by merging warning into longText.

gkrajewski committed:
Item IDs unique 3-digit ordinals

gkrajewski committed:
Empik -> Biedronka
Changed messages & database table name!

gkrajewski committed:
Case contexts instruction corrected

m-krysztofiak reviewed (approved):
Przejrzałam commits, jest ok, ale będę jeszcze sprawdzać w apce :)

m-krysztofiak committed:
minor corrections
feminine form and example

gkrajewski committed:
Email when no vouchers changed in gram pilot
Was projekt.starwords, is multilada

gkrajewski committed:
Grammar instructions final refinements
(i) some suggestions by ZA (ii) more nuanced about child errors and examples used (iii) alternatives intro shortened to be less redundant with instructions repeated on each page

gkrajewski commented:
Dodałem już naprawdę ostatnie dwa commity:

- [fcab799](https://github.com/gkrajewski/CDI-online/pull/312/commits/fcab7999f3bc14d5ed9555d826e3dd2985e180f2)
- [301429c](https://github.com/gkrajewski/CDI-online/pull/312/commits/301429c8b418ebb1c389259af5a08586e5784fca)

gkrajewski committed:
Last minute complexity edits
(1) Typo in instructions (2) Diminutive removed from one item (it could draw away parents attention from syntax)

gkrajewski commented:
Teraz już naprawdę ostatni ([4e784e5](https://github.com/gkrajewski/CDI-online/pull/312/commits/4e784e54de147107c1e37d33aff678e359ac1bc1)). Ważna jest zmiana "autkiem" na "autem", żeby rodzice skupili się na składni (w tych pozycjach smoczych i tak jest za dużo samych składniowych różnic, tzn. więcej niż jedna w parze).
